src/Evolution/Systems/ScalarTensor/Sources/ScalarSource.cpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/ScalarTensor/Sources/ScalarSource.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/SetNumberOfGridPoints.hpp"
+
+namespace ScalarTensor {
+
+void add_scalar_source_to_dt_pi_scalar(
+    gsl::not_null<Scalar<DataVector>*> dt_pi_scalar,
+    const Scalar<DataVector>& scalar_source, const Scalar<DataVector>& lapse) {
+  dt_pi_scalar->get() += get(lapse) * get(scalar_source);
```

Comment:
`get(*dt_pi_scalar)` (fix only if you're fixing other things anyway)

Same in function below

Reply:
I've changed it. There was a typo in the doc so I changed this as well